apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceHeaderContext.cs

```diff
@@ -41,8 +41,9 @@ public sealed class TraceHeaderContext
         /// A regex to match the trace header. 
         /// - ([A-Fa-f0-9]{32}): The trace id, a 32 character hex value.
         /// - ([0-9]+): The span id, a 64 bit integer.
-        /// - (?:;o=([0-3])): The trace mask, 1-3 denote it should be traced. (The ?: makes the outer group non-capturing.)
+        /// - (?:;o=([0-1])): The trace mask, 1 denotes it should be traced. (The ?: makes the outer group non-capturing.)
         /// </summary>
+        /// <remarks>See here for format information: https://cloud.google.com/trace/docs/setup#force-trace. </remarks>
         internal static readonly Regex TraceHeaderRegex =
             new Regex(@"^([A-Fa-f0-9]{32})/([0-9]+)(?:;o=([0-3]))?$", RegexOptions.Compiled);
```

Comment:
Any reason we don't want to change the regex? We could then just check whether `match.Groups[3].Value == "1"` rather than bother parsing it, too.

Reply:
None at all, this just slip through, I changed the comment and didn't change the regex duh. On it.

Reply:
Done, I'm rerunning integration tests locally again, but PTAL.